editor.planx.uk/src/ui/editor/ListManager/ListManager.stories.tsx

```diff
@@ -60,4 +60,24 @@ export const Basic = () => {
   );
 };
 
+export const WithinTemplatedNode = () => {
+  const [tasks, setTasks] = useState<Array<Item>>([
+    { name: "PlanX primary", color: "#0010A4" },
+    { name: "GOV.UK focus", color: "#ffdd00" },
+  ]);
+
+  return (
+    <ListManager
+      values={tasks}
+      onChange={setTasks}
+      Editor={Editor}
+      newValue={() => ({
+        name: "",
+        color: "#fff",
+      })}
+      isTemplatedNode={true}
+    />
+  );
+};
+
```

Comment:
In the future, it'll be much more ideal to introduce more editor tests here per component with user roles mocked - but Storybook is a nice interim / MVP option for demonstrating how we expect the `ListManager` to render within a fully-editable versus templated node:
- Re-order handle hidden
- Bin deletion icon hidden
- Insert item between existing items `+` button hidden
- `add new` button hidden
![Screenshot from 2025-06-24 13-31-29](https://github.com/user-attachments/assets/c11fd1dc-2def-4f61-86d4-7b3ca968ba90)